### PR TITLE
feat: Vote round number 3

### LINKED ISSUE #34: feat: voting round 3
Use this as an example:

https://docs.google.com/spreadsheets/d/1YmjQaf7_9wvfsLMkAF5L1GNv0nXFXtMeTsc2CBsyQ6s/edit#gid=0

### PR DESCRIPTION
**Pull request by SHAKOTN**
## Wtf is this
https://docs.google.com/spreadsheets/d/1YmjQaf7_9wvfsLMkAF5L1GNv0nXFXtMeTsc2CBsyQ6s/edit#gid=0 < use this as an example for voting

## Pls explain simple
1. Each pool votes for 90% of it's graviAURA balance for itself. The only exception is auraBAL pool. Check 3rd point
2. 10% is taken as fees and votes for ~bribes~ badger/wbtc pool for now. This is a subject to change very soon
3. auraBAL pool falls under council regulation, we have to subtract 33% of it's graviAURA and vote for Booba as wstETH pool to support BAL ecosystem

Fixes #34

### CODE REVIEW

File: aura_voter/constants.py [hunk 1]
```diff
@@ -65,3 +65,20 @@
...
+
+# Snapshot pool name mappings
+AURABAL_WETH_GRAVIAURA = "33/33/33 auraBAL/graviAURA/WETH"
+META_WSTETH_WETH = "MetaStable wstETH/WETH"
+BOOBA_TRI_STABLE = "bb-a-USDT/bb-a-DAI/bb-a-USDC"
+POOL_ID_TO_NAME_MAP = {
+    '0x0578292cb20a443ba1cde459c985ce14ca2bdee5000100000000000000000269': AURABAL_WETH_GRAVIAURA,
+    '0xb460daa847c45f1c4a41cb05bfb3b51c92e41b36000200000000000000000194': BADGER_WBTC_POOL_NAME,
+    # TODO: Add real name for digg pool once it's out on Snapshot
+    '0x8eb6c82c3081bbbd45dcac5afa631aac53478b7c000100000000000000000270': "DIGG",
```
Comment (SHAKOTN) [inline aura_voter/constants.py:77]: To change once we know DIGG snapshot voting name tomorrow

File: aura_voter/tests/test_algorithms/test_poc_algorithm.py [hunk 2]
```diff
@@ -2,40 +2,41 @@
...
     votes = voter.propose_voting_choices()
-    assert votes == {
-        BADGER_WBTC_POOL_NAME: Decimal('99.27999999999999998223643161'),
-        'some_pool1': Decimal('0.09000000000000000222044604925'),
-        'some_pool2': Decimal('0.2700000000000000066613381478'),
-        'some_pool3': Decimal('0.3600000000000000088817841970')
-    }
+    assert votes == {'33/33/33 auraBAL/graviAURA/WETH': Decimal('34.82476162062310654454978017'),
+                     '80/20 BADGER/WBTC': Decimal('18.27195247242537317520887355'),
+                     'DIGG': Decimal('27.84495865188414766986031316'),
```
Comment (mitche50) [inline aura_voter/tests/test_algorithms/test_poc_algorithm.py:37]: Just a note, will need to update this as well once DIGG is named

File: aura_voter/voting_algorithms/poc_algorithm.py [hunk 3]
```diff
@@ -32,17 +49,30 @@ def propose_voting_choices(self) -> Dict[str, Decimal]:
...
-            # pool
-            finalized_votes[pool_name] = ((
-                self.ALGORITHM_SETTINGS.badger_pools_fixed_vote_weight * pool_balance
+            pool_id = pool.pool_id
+            if self.ALGORITHM_SETTINGS.regulations.get(pool_id):
+                # Decision made by council to subtract Y% as regulation on auraBAL pool
+                regulation_taken_voting_power += (
+                    pool.balance_aura * self.ALGORITHM_SETTINGS.regulations[pool_id]
+                )
+                voting_power = pool.balance_aura - regulation_taken_voting_power
```
Comment (mitche50) [inline aura_voter/voting_algorithms/poc_algorithm.py:60]: Wouldn't this compound for multiple pools?  Second pool would have the balance_aura from pool1 + balance_aura from pool2 subtracted.

**Review by mitche50 — COMMENTED** [review]
One comment and `propose_voting_choices_stable` is a dead function so can be removed

File: aura_voter/voting_algorithms/poc_algorithm.py (hunk 3 above)
Comment (SHAKOTN) [reply aura_voter/voting_algorithms/poc_algorithm.py]: Should be tackled now

**Review by mitche50 — APPROVED** [review]

**Pull request merged**